bind/mount.go

```diff
@@ -201,3 +198,97 @@ func leaveBindMountAlone(mount specs.Mount) bool {
 	}
 	return false
 }
+
+// RecursiveUnmount unmounts the given mountpoints and anything that's hanging
+// off of them, rather aggressively.  If a mountpoint also appears in the
+// mountpointsToRemove slice, the mountpoints are removed after they are
+// unmounted.
+func RecursiveUnmount(mountpoint string, mountpointsToRemove []string) error {
```

Comment:
nit on the name, not really recursive is it?  FullUnmount, UmountAll, HardUmount, or something similar instead?

Reply:
The function isn't recursive, true.  The name mirrors that of the similar function in the `github.com/containers/storage/pkg/mount` package, which does something similar, but which selects the order in which it unmounts things by sorting the directory paths of the mountpoints, as we did previously, and which doesn't detach busy filesystems.  If you have a preference for a different name, I'm happy to change it.

Reply:
Would UnmountMountpoints work for you?

Reply:
Sure.  Making the change.